### PR TITLE
feat(client): design homepage and FAB

### PR DESCRIPTION
**Pull request by nihalanisumit**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by nihalanisumit** [comment]
fab

**Comment by nihalanisumit** [comment]
homepage

File: client/src/components/FloatingActionButton/FloatingActionButton.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
```
Comment (nimish-gupta) [inline client/src/components/FloatingActionButton/FloatingActionButton.jsx:2]: you have removed react-router-dom package from the package.json in this PR, will this throw an error?

**Review by nimish-gupta — APPROVED** [review]

File: client/src/components/FloatingActionButton/FloatingActionButton.jsx (hunk 1 above)
Comment (nihalanisumit) [reply client/src/components/FloatingActionButton/FloatingActionButton.jsx:2]: @nimish-gupta while resolving conflict the package was at two locations in package.json so removed one of them

**Comment by karansapolia** [comment]
@nihalanisumit  One use case issue with scrolling:
Use Case Gif

**Review by karansapolia — APPROVED** [review]
Only the mentioned use-case, otherwise LGTM

**Review by piyushagwl007 — APPROVED** [review]

**Pull request merged**